tools/IoTEdgeDevOps/DevOpsLibTest/ReleaseManagementTest.cs

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DevOpsLibTest
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using DevOpsLib;
+    using DevOpsLib.VstsModels;
+    using Flurl;
+    using Flurl.Http.Testing;
+    using NUnit.Framework;
+
+    public class ReleaseManagementTest
+    {
+        const string PersonalAccessToken = "pattoken999";
+        ReleaseManagement releaseManagement;
+        HttpTest httpTest;
+
+        [SetUp]
+        public void Setup()
+        {
+            this.httpTest = new HttpTest();
+            this.releaseManagement = new ReleaseManagement(new DevOpsAccessSetting(PersonalAccessToken));
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            this.httpTest.Dispose();
+        }
+
+        [Test]
+        public async Task TestGetReleasesAsyncWithEmptyBranchNameAsync()
```

Comment:
What about the case when the `branchName` contains JSON escape characters?

Reply:
This test is not testing this; and I don't think JSON escape characters are problem when de-serialization.  If you have concerns, please add suggestion to one of release management test to change branch name with escape character in test data.

Reply:
The key is I'm not sure how Flurl deal with a GET/POST parameter that has an escaped JSON.  Right now we kinda assume that it would handle the escape properly.

Reply:
We can test whether Flurl with escaped JSON; I think all data will be uri-encoded therefore it should works exactly the same.  Also what escape character do you think it will happen in our scenario.  Can you give me an example?